sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/EditPersonSteps.java

```diff
@@ -441,6 +442,13 @@ public EditPersonSteps(
           softly.assertTrue(elementVisible, "Travel Entry isn't visible");
           softly.assertAll();
         });
+    And(
+        "I check General comment field",
+        () -> {
+          Assert.assertTrue(
+              webDriverHelpers.isElementEnabled(GENERAL_COMMENT_FIELD),
+              "There is no resizable General comment field on page");
```

Comment:
we can remove resizable since we don't test drag&drop actions over components

Reply:
@rdutu-vg I am checking if field is resizable by v-textarea-resizable parameter is in class.